pymagicc/definitions/__init__.py

```diff
@@ -0,0 +1,17 @@
+from os.path import dirname, join
+
```

Comment:
This module will hold all of our MAGICC definitions. It's useful for writing files as well as for unit conversions later on.